### PR TITLE
Mistake

### PR DESCRIPTION
**Pull request by xakimovotabek200**

### CODE REVIEW

**Review by tolibjon1015 — APPROVED** [review]

**Review by Ahmadullo21 — APPROVED** [review]

File: src/components/Cards/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                      </div>
+                    </div>
+                  </div>
+                </section>
+              </SwiperSlide>
+            );
+          })}
+        </Swiper>
+      </div>
+    </div>
```
Comment (tolibjon1015) [inline src/components/Cards/Card.jsx:137]: responsive to'g'irlang

**Review by tolibjon1015 — DISMISSED** [review]

File: src/components/Cards/Card.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+.item:hover .icon {
+  color: white;
+  background-color: #f16262;
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  display: block;
+  transition: all 0.5s ease-in-out;
+}
```
Comment (ibrohim5009) [inline src/components/Cards/Card.css:17]: nimaga css tailwend emas ?????????????????????????????

Comment (akbarali0705) [inline src/components/Cards/Card.css:17]: Css nima qilyapti?

**Review by ibrohim5009 — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  height: 100%;
+}
+
+.swiper-slide {
+  text-align: center;
+  font-size: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (akbarali0705) [inline src/style.css:16]: Css?

**Review by akbarali0705 — CHANGES_REQUESTED** [review]

File: src/components/Cards/Card.jsx (hunk 1 above)
Comment (Ahmadullo21) [reply src/components/Cards/Card.jsx:137]: responsive to'g'irlang

**Review by Ahmadullo21 — CHANGES_REQUESTED** [review]

File: src/components/Cards/Card.css (hunk 2 above)
Comment (ibrohim5009) [inline src/components/Cards/Card.css:17]: nimaga css?

**Review by ibrohim5009 — CHANGES_REQUESTED** [review]

File: src/components/Cards/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        },
+        "(min-width: 1000px)": {
+          slides: { perView: 4, spacing: 10 },
+        },
+        "(max-width: 800px)": {
+          slides: { perView: 1, spacing: 10 },
+        },
+      },
+      loop: true,
+    },
```
Comment (samandar0806) [inline src/components/Cards/Card.jsx:24]: React App - Google Chrome 3_24_2023 7_33_28 AM

nimaga (love) button hoverda ozgarib ketyapti

**Review by samandar0806 — APPROVED** [review]

**Review by ibrohim5009 — APPROVED** [review]

File: src/data.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    price: "10% off",
+    Image: Img2,
+    icon: <i class="fa-regular fa-heart"></i>,
+    text: {
+      h3: "Gears Xbox Controller",
+      p: "$30.000",
+      del: "$35.000",
+    },
+  },
+];
```
Comment (akbarali0705) [inline src/data.js:129]: image
look at responsive

**Review by akbarali0705 — CHANGES_REQUESTED** [review]

**Review by akbarali0705 — APPROVED** [review]

**Review by ibrohim5009 — DISMISSED** [review]
media ???

**Review by tolibjon1015 — CHANGES_REQUESTED** [review]

**Review by akbarali0705 — APPROVED** [review]

**Review by akbarali0705 — APPROVED** [review]

File: src/components/Cards/Card.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+.card:hover .icon {
+  color: white;
+  background-color: #f16262;
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  display: block;
+  transition: all 0.5s ease-in-out;
+}
```
Comment (tolibjon1015) [inline src/components/Cards/Card.css:17]: image
orasidagi masofa juda katta va cardga hover bo'lganda sekin chiqib tez orqaga qaytib ketyapti !!!

**Review by tolibjon1015 — DISMISSED** [review]

File: src/components/Cards/Card.css (hunk 6 above)
Comment (xakimovotabek200) [reply src/components/Cards/Card.css:17]: sandayam shu muammmo

**Review by Ahmadullo21 — APPROVED** [review]

**Comment by ibrohim5009** [comment]
image
media ???????????

**Review by ibrohim5009 — CHANGES_REQUESTED** [review]
src/assepts/PubgSectionImg/game-action.png

**Review by ibrohim5009 — APPROVED** [review]

**Review by Ahmadullo21 — APPROVED** [review]

**Pull request merged**